eth/tracers/native/prestate.go

```diff
@@ -74,7 +75,7 @@ type prestateTracerConfig struct {
 	DiffMode bool `json:"diffMode"` // If true, this tracer will return state modifications
 }
 
-func newPrestateTracer(ctx *tracers.Context, cfg json.RawMessage) (*tracers.Tracer, error) {
+func newPrestateTracer(ctx *tracers.Context, cfg json.RawMessage, _ *params.ChainConfig) (*tracers.Tracer, error) {
```

Comment:
Please don't use unnamed params. It  was an unfortunate language design that the authors also think is bad now but can't revert it. Still, live code should not use it, it just makes things harder to understand and later modify if the param is ever needed.